prisma/schema.prisma

```diff
@@ -67,3 +67,18 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+// Profile schema
+enum Role {
+  ADMIN
+  MENTOR
+  MENTEE
+}
+
+model Profile {
+    user            User    @id @unique
+    preferred_name  String?
```

Comment:
Take a look at the solution to https://github.com/prisma/prisma/discussions/2530. Not sure if we want to have to use snake_case names in the code when we could be using camelCase and just use snake_case for the database.